index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Intento de YouTube</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav id="topNav">
+        <img id="logo" src="./imagenes/logo.jpg" alt="">
+        <div>
+        <input id="search" placeholder="search" type="text">
+        <button>search</button>
+        </div>
+        <div id="profile">
+        <button>notifications</button>
+        <button id="profile">profile</button>
+        </div>
+        
+    </nav>
+    <div id="main">
+        <div id="left">
+    <video controls> 
+        <source src="./videos/ExamenFinal.mp4" type="video/mp4">
+    </video>
+    <div id="title">
+        <h1>Examen?</h1>
+        <div id="like">
+        <button>like</button>
+        <button>don't like</button>
+        <button>share</button>
+        </div>
+    </div>
+    <h3>21035 views      3/03/2022</h3>
+    <h3>Siempre que pienso en la universidad</h3>
+    <input placeholder="Comments" id="comment" type="text">
+    </div>
+    <div id="right">
+        <div id="siguiente">
+        <img src="./imagenes/simpsons.jpg" alt="">
+        <p>Los Simpsons video 1</p>
+        </div>
+        <div id="siguiente">
+        <img src="./imagenes/simpsons1.jpg" alt="">
+        <p>Los Simpsons video 2</p>
+        </div>
+        <div id="siguiente">
+        <img src="./imagenes/simpsons2.jpg" alt="">
+        <p>Los Simpsons video 3</p>
+        </div>
+        <div id="siguiente">
+        <img src="./imagenes/simpsons3.jpg" alt="">
+        <p>Los Simpsons video 4</p>
+        </div>
+    </div>
```

Comment:
you should use a class tag instead of an Id tag

Reply:
It would be better if you explained why this change is required.

Reply:
I recommend you change the property ID to a class because you can't repeat the same ID for different parts of the code